testing/opencolorio/APKBUILD

```diff
@@ -0,0 +1,69 @@
+# Contributor: Mark Riedesel <mark@klowner.com>
+# Maintainer:
+pkgname=opencolorio
+pkgver=1.0.9
+pkgrel=0
+pkgdesc="A color management framework for visual effects and animation"
+url="http://opencolorio.org"
+arch="all"
+license="BSD"
+depends=""
+makedepends="cmake python2-dev python3-dev freeglut-dev glew-dev"
+install=""
+subpackages="$pkgname-python2:py2 $pkgname-python3:py3 $pkgname-dev $pkgname-utils"
+source="$pkgname-$pkgver.tar.gz::http://github.com/imageworks/OpenColorIO/archive/v$pkgver.tar.gz"
+builddir="$srcdir"/OpenColorIO-$pkgver
+
+build() {
+	cd "$builddir"
+
+	for pyver in python3 python2; do
+		mkdir build-$pyver && cd build-$pyver
+
+		cmake .. \
+			-DCMAKE_INSTALL_PREFIX=/usr \
+			-DPYTHON=/usr/bin/$pyver \
+			-DOCIO_BUILD_TRUELIGHT=OFF \
+			-DOCIO_BUILD_NUKE=OFF \
+			|| return 1
+		make
```

Comment:
Okay, I had to be more accurate previously. Every statement inside a function that may fail (and should not) should be followed by `|| return 1` (until we change abuild to run `set -e`). It’s not needed after the last statement of a function, because shell has implicit return. For example `cd ..`, `mkdir -p` etc. normally don’t fail, so not needed here, but `make` can fail and it should halt the built in that case.